### PR TITLE
Change pictures in CareerSection, title in TeamSection and add Copyright

### PR DESCRIPTION
**Pull request by Grappler2010**
1. Change pictures in CareerSection.
2. Change title in TeamSection.
3. Add copyright in footer.
4. Fix second video size.
5. Fix logo sizes.

### CODE REVIEW

File: app/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
 export const Footer = () => {
+  const { isMobile } = useWindowSize();
+
   return (
     <Container>
       <footer className={styles.component}>
-        <Logo className={styles.footerLogo} />
-        <Contacts />
+        {
+          isMobile ?
```
Comment (dealwith) [inline app/components/Footer/Footer.tsx]: please change this logic to use css

Comment (Grappler2010) [reply app/components/Footer/Footer.tsx]: Done

File: app/components/Logo/logo.module.scss [hunk 2]
```diff
@@ -2,6 +2,9 @@
 @use "styles/breakpoints";
 
 .logo {
+  width: 180px;
+  height: 50px;
+
   @include mixin.media-w(breakpoints.$laptop) {
     width: 96px;
```
Comment (dealwith) [inline app/components/Logo/logo.module.scss:9]: I guess that height of the logo should also be smaller on mobile, right?

**Review by dealwith — APPROVED** [review]
Please check my commit to logo on mobile, and it's good to go for me 👍

File: app/components/Logo/logo.module.scss (hunk 2 above)
Comment (Grappler2010) [reply app/components/Logo/logo.module.scss:9]: Add logo height for mobile. Thanks for the comments

**Pull request merged**